assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -32,14 +33,15 @@ che.workspace.storage=${che.home}/workspaces
 # Your projects are synchronized from the Che server into the machine running each
 # workspace. This is the directory in the machine where your projects are placed.
 che.workspace.projects.storage=/projects
+
 # Used when devfile k8s/os type components requests project PVC creation
 # (applied in case of unique and perWorkspace PVC strategy. In case of common PVC strategy,
 # it will be rewritten with value of che.infra.kubernetes.pvc.quantity property)
 che.workspace.projects.storage.default.size=1Gi
 
-#Defines the directory inside the machine where all the workspace logs are placed.
-#The value of this folder should be provided into machine e.g. like environment variable
-#so agents developers can use this directory for backup agents logs.
+# Defines the directory inside the machine where all the workspace logs are placed.
+# The value of this folder should be provided into machine e.g. like environment variable
+# so agents developers can use this directory for backup agents logs.
```

Comment:
Is this here solely to unify the format or does the parser tolerate require single space after `#`? Because `che.properties` is hand-written, I'd be slightly nervous to require more strict format than the properties file dictates.

Reply:
yes it's beter to have spaces since `#`-s are stripped so to avoid words/sentences gluing each other.